vega-embed/README.md

```diff
@@ -2,9 +2,9 @@
 
 [](dependency)
 ```clojure
-[cljsjs/vega-embed "3.1.1-0"] ;; latest release
-[cljsjs/vega "3.0.1-0"] ;; latest release
-[cljsjs/vega-lite "2.0.0-0"] ;; latest release
+[cljsjs/vega-embed "3.16.1-0"] ;; latest release
+[cljsjs/vega "3.3.1-0"] ;; last B4 4.x release
```

Comment:
Actually only the `cljsjs/vega-embed` line is needed here. As `cljsjs/vega` and `cljsjs/vega-lite` are listed as dependencies of `cljsjs/vega-embed`, they're automatically pulled in and you do not need to explicitly list them in the dependencies.

Thus please remove them from this list.

Reply:
OK, they are removed - and new README committed. I will make new pull request